bmwqemu.pm

```diff
@@ -69,16 +69,16 @@ sub result_dir { 'testresults' }
 use constant STATE_FILE => 'base_state.json';
 
 # Write a JSON representation of the process termination to disk
-sub serialize_state {
-    my $state = {@_};
+sub serialize_state (%args) {
+    my $state = {%args};
```

Comment:
```suggestion
sub serialize_state (%state) {
```

And then use `%state` directly, e.g. `delete $state{error}` and `encode_json(\%state)`.

Reply:
done